scripts/create_ids_augmented.py

```diff
@@ -0,0 +1,82 @@
+import pandas as pd
+import numpy as np
+import pyalex
+from pathlib import Path
+import tomli
+
+# globals
+OPENALEX_MAX_OR_LENGTH = 100
+OPENALEX_MAX_PAGE_LENGTH = 200
+OPENALEX_PREFIX = "https://openalex.org/"
+SEED = 535
+
+
+def abs_length_ok(row):
+    return len(str(row["abstract"]).split()) >= 20
```

Comment:
Character count here as well I think :) because we will do it after the fact now, but ideally it would have been done here